src/smf/gn-handler.c

```diff
@@ -93,8 +93,8 @@ uint8_t smf_gn_handle_create_pdp_context_request(
         ogs_error("No SGSN Address for user traffic");
         cause_value = OGS_GTP1_CAUSE_MANDATORY_IE_MISSING;
     }
-    if (req->msisdn.presence == 0) {
-        ogs_error("No MSISDN");
+    if (req->imsi.presence == 0 && req->msisdn.presence == 0) {
```

Comment:
Regarding GTPv1C (Gn), see the info I provided in https://github.com/open5gs/open5gs/pull/1519#issuecomment-1116321292
It seems in GTPv1C this IE is always expected, even if it may take a dummy value. (It is always expected if no secondary pdp context, but we don't support that anyway).

Reply:
You are right. I will revert this part. Thanks for clarification :)